metals/src/main/scala/scala/meta/internal/metals/codeactions/CreateCompanionObjectCodeAction.scala

```diff
@@ -68,6 +69,22 @@ class CreateCompanionObjectCodeAction(
 
   }
 
+  def endPosOfTree(t: Tree): Position = {
```

Comment:
I think it would be more readable if this returns `l.Position` to truly reflect the end position of the tree.

Reply:
In this case this is `scala.meta.inputs.Position`.
We use `l` prefix(`import org.eclipse.{lsp4j => l}`) for lsp structures.

Reply:
oh, right! changed